app/electron/main.js

```diff
@@ -127,6 +128,9 @@ function createWindow () {
   let serverFilePath;
   if (!isDev) {
     serverFilePath = path.join(process.resourcesPath, './electron/server');
+    if (isWin) {
+      execSync(`ren ${serverFilePath} server.exe`)
+    }
```

Comment:
I don't think this is the place for this. We'd be running this as an external command and on every run of the app.
Ideally we should compile the server to have that extension on Windows already (as it seems to be needed), so it's something I'd check to see how it could be added in the backend, not the electron app.